app/src/test/java/com/globant/onboardingcalculatorwithkotlin/CalculatorPresenterTest.kt

```diff
@@ -23,6 +23,7 @@ import com.nhaarman.mockitokotlin2.verify
 import org.junit.Test
 import org.junit.Assert.assertEquals
 
+const val WRONG_OPERATOR: Char = '?'
```

Comment:
if this is not used outside the test, it should be inside the companion object

Reply:
Done!